rdata/parser/_xdr.py

```diff
@@ -15,14 +15,9 @@ class ParserXDR(Parser):
     def __init__(
         self,
         data: memoryview,
-        *,
-        expand_altrep: bool = True,
-        altrep_constructor_dict: AltRepConstructorMap = DEFAULT_ALTREP_MAP,
+        **kwargs: Any,
```

Comment:
I added the parameters back (d1934507202b040a4ab226b807d363d327c34993), but without default values so that they are not duplicated from `_parser.py` here.